src/r2mm/installing/profile_installers/GenericProfileInstaller.ts

```diff
@@ -181,10 +181,24 @@ export default class GenericProfileInstaller extends ProfileInstallerProvider {
         }
     }
 
+    private async installGodotML(mlLocation: string, modLoaderMapping: ModLoaderPackageMapping, profile: Profile) {
+        const copyFrom = path.join(mlLocation, "addons", "mod_loader");
```

Comment:
Do we not want to copy the addons folder entirely?

Reply:
I was considering this from the perspective of cleanup (if cleanup would be necessary), so I figured it's more appropriate to be explicit about the structure as it makes cleanup at least possible if it becomes necessary. Otherwise we'd have to delete all the addons or implement some state tracking.